tests/resources/test_resources.py

```diff
@@ -90,16 +91,33 @@ def test_simple_flow(mock_apply_async, app, db, client, user):
     assert res.status_code == 200
     assert res.json == expected_job
 
-    # Create/trigger a run
+    # Create/trigger a run with invalid `since` to test correct failure
+    invalid_since_arg = "2025-05-20T00:00:00+aa:bb"  # slightly invalid ISO timestamp
+    res = client.post(
+        f"/jobs/{job_id}/runs",
+        json={
+            "title": "Manually triggered run",
+            "args": {
+                "since": invalid_since_arg,
+                "job_arg_schema": "PredefinedArgsSchema",
+            },
+            "queue": "celery",
+        },
+    )
+    assert res.status_code == 400
+
+    # Create/trigger a run with valid inputs
+    since_arg = (datetime.now(timezone.utc) - timedelta(days=2)).isoformat()
     res = client.post(
         f"/jobs/{job_id}/runs",
         json={
             "title": "Manually triggered run",
-            "args": {},
+            "args": {"since": since_arg, "job_arg_schema": "PredefinedArgsSchema"},
```

Comment:
by replacing the args here, we are losing the test case for a job without arguments. Could we have both?